nbs/12_cli.ipynb

```diff
@@ -49,6 +49,7 @@
    "outputs": [],
```

Comment:
<p>Some of these are inferred from git but some are inferred from GitHub perhaps the distinction is important as that could confuse some users regarding the origin of some of the values and where they are inferred from </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>